components/renderers/admin/AdminRenderer.js

```diff
@@ -32,6 +33,10 @@ const AdminRenderer = ({ type, ...params }) => {
       return <VideoRenderer accept={validVideoTypes} {...params} />;
     case CMS_TYPES.BOOLEAN:
       return <BooleanRenderer type={type} {...params} title="Yes" />;
+    case CMS_TYPES.RICH_TEXT:
+      const richTextAttributeType = new RichTextAttributeType();
+      const RichTextComponent =  richTextAttributeType.editableComponent();
+      return <RichTextComponent {...params} {...richTextAttributeType.props()} />;
```

Comment:
In the future, all data types would pass this code. The switch part of AdminRendderer would be deprecated:

```suggestion
      const attributeType = AttributeTypeFactory.create({metadata: {type, custom_name: params.customName});
      const Component =  attributeType.editableComponent();
      return <Component {...params} {...attributeType.props()} />;
```